x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/findings_flyout.tsx

```diff
@@ -143,6 +158,19 @@ export const FindingsRuleFlyout = ({ onClose, findings }: FindingFlyoutProps) =>
       <EuiFlyoutBody key={tab.id}>
         <FindingsTab tab={tab} findings={findings} />
       </EuiFlyoutBody>
+      <EuiFlyoutFooter>
+        <EuiFlexGroup gutterSize="none" justifyContent="flexEnd">
+          <EuiFlexItem grow={false}>
+            <EuiPagination
+              aria-label={PAGINATION_LABEL}
+              pageCount={findingsCount}
+              activePage={flyoutIndex}
+              onPageClick={onPaginate}
+              compressed
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlyoutFooter>
```

Comment:
I think @tinnytintin10 asked that this pagination bar would be on the top left corner.
It's also important that we have the close button in the same place in all of our flyouts, preferably also the same placement as the rest of kibana to keep the same ux.

Reply:
I am fine with this placement. :ship_it:

Reply:
🎉🚀